### PR TITLE
Update everything

### PR DESCRIPTION
**Pull request by elegaanz**
I proofread all the pages and updated was needed to be updated.

### CODE REVIEW

File: source/CLI.html.md [hunk 1]
```diff
@@ -101,7 +117,7 @@ plm search unlock -p Plume
 
 **Arguments:**
 
-- `--path`, `-n`: path to plume working directory.
+- `--path`, `-n`: path to Plume's working directory.
```
Comment (trinity-1686a) [inline source/CLI.html.md]:
```suggestion
- `--path`, `-p`: path to Plume's working directory.
```

**Review by trinity-1686a — COMMENTED** [review]
this bit is false I believe, and if it's not, we should make it

File: source/CLI.html.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ summary: 'plm is a little CLI tool that can help you to manage your instance fro
 command line if you are admin of an instance.'
 ---
 
+You can see these help messages with `plm --help` or `plm COMMAND --help` (where `COMMAND` is the specific subcommand you are interested in).
+
 If any required argument is ommitted, you will be asked to input manually.
 
 ## `plm instance`
```
Comment (igalic) [inline source/CLI.html.md:12]: insert a line break here

different markdown engines will not treat this as header otherwise

**Review by igalic — APPROVED** [review]
👍

File: source/CLI.html.md (hunk 2 above)
Comment (elegaanz) [reply source/CLI.html.md:12]: I'm not sure of where I should add the line break, there already are blank lines before and after the heading? (and both GitHub and Middleman's engines render it correctly apparently, so…)

Comment (igalic) [reply source/CLI.html.md:12]: before and after, yes

also, sorry, this looked very different on my phone in the dark, to extremely tired eyes 😩

**Pull request merged**